### DISCUSSION TITLE
Song select audio track fails to change when switching between beatmaps created after 2022.621.0

### DISCUSSION THREAD

**Discussion by Joppe27** [ppy/osu#19031] [Q&A] [answered]
1. Create a beatmap with any audio file
2. Create a second beatmap with another audio file

The audio track will do nothing when switching between these maps in song select: the audio for beatmap 1 will keep playing. This bug even affects the editor, which makes it possible to edit a beatmap with an incorrect song loaded (see video).

Or in video form: https://joppe27.s-ul.eu/aVDmoVCL

Cause is likely https://github.com/ppy/osu/pull/18703.

**Comment by peppy** [comment, marked as answer]
This means that both beatmaps actually do have the same audio track internally.

~~What's the process you're following when making the new beatmaps?~~

I can reproduce this, all good.

**Reply by peppy** [reply]
Tracking via https://github.com/ppy/osu/issues/19033.
